src/extra/normalize.src.css

```
@@ -130,14 +130,15 @@
 :where(select) {
   padding-inline: var(--size-relative-4) 0;
   padding-block: .75ch;
+  field-sizing: content;
 }
 
 :where(textarea, select, input:not([type="button"],[type="submit"],[type="reset"])) {
   background-color: var(--surface-2);
   border-radius: var(--radius-2);
 }
 
-:where(textarea) { resize: block }
+:where(textarea) { resize: block; field-sizing: content }
```

```suggestion
:where(textarea) { 
  resize: block;
  field-sizing: content;
  min-inline-size: var(--size-content-1);
  min-block-size: 2lh;
}
```

this ensures the textarea doesnt simply collapse once field-sizing: content is applied.